packages/repository-json-schema/src/build-schema.ts

```diff
@@ -501,7 +502,11 @@ export function modelToJsonSchema<T extends object>(
       result.properties = result.properties ?? {};
       const relMeta = meta.relations[r];
       const targetType = resolveType(relMeta.target);
-      const targetSchema = getJsonSchema(targetType, options);
+      // omit the title for related schema
+      const targetSchema = getJsonSchema(
+        targetType,
+        _.omit(options, ['title']),
```

Comment:
The `omit` function can be implemented as:

```ts
function omit(options: object, ...props: string[]} {
  const copy = {...options};
  for (const p of props) {
    delete copy[p];
  }
  return copy;
}

Then just call `omit(options, 'title')`

Reply:
Let's not add `lodash` as a dependency unless necessary.

Reply:
+1 to avoiding lodash dependency.

I think this line can be also rewritten as follows:

```suggestion
        {...options, title: undefined},
```